scripts/play.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -eu
+
+REPO_ROOT="$(git rev-parse --show-toplevel)"
+cd $REPO_ROOT/src
```

Comment:
I recognise this pattern :D 

you probably want to make this script executable. run `chmod u+x scripts/play.sh` and then `git commit scripts/play.sh`.